### PR TITLE
2021 projects

### PR DESCRIPTION
**Pull request by jzhzhang66**
This PR contains code to update the archive to contain 2021 projects.

### CODE REVIEW

**Review by zefwang — APPROVED** [review]
It feels like I wasn't even at HBP21 but LGTM

**Review by ghost — APPROVED** [review]
LGTM! after you merge, check to make sure the build succeeds, and verify that your changes show up properly

thanks judy <3

**Pull request merged**
